dask_cloudprovider/cloudprovider.yaml

```diff
@@ -30,3 +30,15 @@ cloudprovider:
     environment: {} # Environment variables that are set within a task container
     find_address_timeout: 60 # Configurable timeout in seconds for finding the task IP from the cloudwatch logs.
     skip_cleanup: False # Skip cleaning up of stale resources
+  azure:
+    experiment_name: "dask-cloudprovider" # default name of the Experiment to submit
+    initial_node_count: 1 # Initial node count
+    jupyter: False # by default start the Jupyter lab instance on the headnode
+    jupyter_port: 9000 # default Jupyter lab port on the headnode
+    dashboard_port: 9001 # default Dask dashboard port on the headnode
+    scheduler_port: 9002 # default for the scheduler port if using from outside of cluster VNET
+    additional_ports: [] # list of tuples of additional ports to map/forward
+    admin_username: "" # username to log in to the AzureML Training Cluster for 'local' runs
+    admin_ssh_key: "" # password to log in to the AzureML Training Cluster for 'local' runs
+    datastores: [] # default list of datastores to mount
+    code_store: null # required only if datastores are defined
```

Comment:
should remove this (code_store)

Reply:
Agree. Will remove and check in.

Reply:
done, PR to your fork

Reply:
https://github.com/drabastomek/dask-cloudprovider/pull/76